tests/phpunit/api/v3/AddressTest.php

```diff
@@ -436,4 +436,43 @@ public function testCreateAddressStateProvinceIDCorrectForCountry() {
     $this->assertEquals('3497', $address2['values'][0]['state_province_id']);
   }
 
+  public function getSymbolicCountryStateExamples() {
+    return [
+      // [mixed $inputCountry, mixed $inputState, int $expectCountry, int $expectState]
+      0 => [1228, 1004, 1228, 1004],
+      1 => ['US', 'CA', 1228, 1004],
+      2 => ['US', 'TX', 1228, 1042],
+      3 => ['US', 'California', 1228, 1004],
+      4 => [1228, 'Texas', 1228, 1042],
+      // Don't think these have been supported?
+      // 5 => ['United States', 1004, 1228, 1004] ,
+      // 6 => ['United States', 'TX', 1228, 1042],
```

Comment:
One nitpick - there's really no reason to put the array keys 0, 1, 2, etc. in here as php will do it for you if you omit them.

Reply:
added commit to remove the explicit numeric keys

Reply:
There is a reason - when reporting failures, PHPUnit identifies the test case by row number. With unnumbered arrays, I've often found it annoying to determine which row it's talking about. With the numbers, it's easy.

Reply:
Ahhh